seed/migrations/0121_update_updated_timestamps.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+from django.db import connection, migrations
+
+
+def forwards(apps, schema_editor):
+    property_sql = (
+        'UPDATE seed_propertystate '
+        'SET updated = (SELECT created '
```

Comment:
This and the tax lot version of this change looks good - thanks for making it Alex!